src/features/submission/components/submission-ingredient/index.tsx

```diff
@@ -41,30 +36,31 @@ const SubmissionIngredient = ({
     >
       <Skeleton active loading={loading}>
         <StyledCardGrid hoverable={false}>
-          <Typography.Text strong>{songIngredient.song.name}</Typography.Text>
+          <Typography.Text strong>
+            {songIngredient.ingredient.name}
+          </Typography.Text>
         </StyledCardGrid>
         <StyledCardGrid hoverable={false}>
-          {songIngredient.song.difficulty === 'Expert' ? (
-            <ExpertJacket src={songIngredient.song.imageUrl} />
+          {!songIngredient.submitted ? (
+            <StyledIngredient
+              src={`${process.env.ASSETS_URL}${songIngredient.ingredient.image256}`}
```

Comment:
A thought - for future we could have a helper function that creates the entire url from just the latter part. As it is now, if someone uses `www.api.com/` as the ASSETS_URL it will error, rather than `www.api.com`. Another thing to not worry about for now :)